modules/home/assets/js/components/home-screen.js

```diff
@@ -1,13 +1,20 @@
+import { useState } from 'react';
 import { Container, Grid } from '@elementor/ui';
 
 import TopSection from './top-section';
 import BlockSection from './block-section';
 
 const HomeScreen = () => {
+	const baseUrl = elementorAppConfig.pages_url;
+	const [ topScreenProps ] = useState( { videoUrl: 'https://elementor.com/academy/getting-started-with-elementor/', ctaUrl: baseUrl, embedUrl: 'https://www.youtube.com/embed/icTcREd1tAg?si=MPamCEWNeRR_VdAn&amp;controls=0' } );
 	return (
-		<Container sx={ { py: 4 } }>
+    <Container sx={ { py: 4 } }>
```

Comment:
Can you please fix indententation? Something went wrong with the fixing the merge conflicts.

I have introduced a container to the home screen, so that we have a max-width value.

And I have used a grid component, so that we can see gap values.